### ISSUE TITLE
http://hl7.org/fhir/us/saner/FEMADailyHospitalCOVID19Reporting is not a valid coding system for Questionnaire.item

### ISSUE THREAD

**Issue by keithboone** [microsoft-healthcare-madison/learning-spike-erp#20]
"item": [
              {
                "extension": [
                  {
                    "url": "http://fhir.org/guides/argonaut/questionnaire/StructureDefinition/extension-itemOrder",
                    "valueDecimal": 0
                  }
                ],
                "linkId": "collectionDate",
                "code": [
                  {
                    "system": "http://hl7.org/fhir/us/saner/FEMADailyHospitalCOVID19Reporting",
                    "code": "collectionDate"
                  }
                ],

**Comment by GinoCanessa** [comment]
Yeah.  Since I've been testing out the Questionnaires, none of those will exist.  I'll go through the existing sets for Measures (MeasureGroupSystem and MeasurePopulationSystem) and see if I can add matching ones for the Questionnaires.

Don't know if I'll get to it before Monday though.

**Comment by GinoCanessa** [comment]
Added Questionnaire Vocabs in a [PR to the SANER repo](https://github.com/HL7/fhir-saner/pull/19)
